lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -592,6 +609,10 @@ instance ToText StakePoolLog where
             "Writing stake-distribution for epoch " <> pretty ep
         MsgStakePoolRegistration pool ->
             "Discovered stake pool registration: " <> pretty pool
+        MsgUsingRankingEpochConstants csts ->
+            "Using ranking epoch constants: " <> pretty csts
+        MsgUsingTotalStakeForRanking s ->
+            "The total stake used to determine epoch constants was: " <> pretty s
```

Comment:
One last note: this message is displayed at a Notice level when the stake is low. Yet, when reading this message, it's hard to understand _why_ this is a notice and what's wrong with that. What about rephrasing it to something explicit like:

```hs
"The total stake is oddly small (" <> pretty s <> "). This may result in unreliable epoch constants."
```

And, same about the above actually. 

```hs
"The total reward for this is oddly small. This may cause pool's ranking to be unreliable. Epoch constants are:  " <> pretty csts
```